### PR TITLE
Fixed webview caching issue

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

File: CountlyFeedbackWidget.m [hunk 1]
```diff
@@ -45,48 +45,44 @@ + (CountlyFeedbackWidget *)createWithDictionary:(NSDictionary *)dictionary
 - (void)present
 {
     CLY_LOG_I(@"%s", __FUNCTION__);
-
+    
     [self presentWithAppearBlock:nil andDismissBlock:nil];
 }
 
-- (void)presentWithAppearBlock:(void(^ __nullable)(void))appearBlock andDismissBlock:(void(^ __nullable)(void))dismissBlock;
+- (void)presentWithAppearBlock:(void(^ __nullable)(void))appearBlock andDismissBlock:(void(^ __nullable)(void))dismissBlock
```
Comment (arifBurakDemiray) [inline CountlyFeedbackWidget.m:52]: missin semi colon at tje end, if it is not necessary all cool

File: CountlyFeedbackWidget.m [hunk 2]
```diff
@@ -45,48 +45,44 @@ + (CountlyFeedbackWidget *)createWithDictionary:(NSDictionary *)dictionary
...
     webVC.webView = webView;
     NSURLRequest* request = [self displayRequest];
     [webView loadRequest:request];
-
     CLYButton* dismissButton = [CLYButton dismissAlertButton];
     dismissButton.onClick = ^(id sender)
     {
         [webVC dismissViewControllerAnimated:YES completion:^
-        {
+         {
```
Comment (arifBurakDemiray) [inline CountlyFeedbackWidget.m:77]: single whitespace

File: CountlyFeedbackWidget.m [hunk 3]
```diff
@@ -144,25 +140,25 @@ - (void)recordResult:(NSDictionary * __nullable)result
 - (NSURLRequest *)dataRequest
 {
     NSString* queryString = [NSString stringWithFormat:@"%@=%@&%@=%@&%@=%@&%@=%@&%@=%@&%@=%@",
-        kCountlyQSKeySDKName, CountlyCommon.sharedInstance.SDKName,
-        kCountlyQSKeySDKVersion, CountlyCommon.sharedInstance.SDKVersion,
-        kCountlyFBKeyAppVersion, CountlyDeviceInfo.appVersion,
-        kCountlyFBKeyPlatform, CountlyDeviceInfo.osName,
-        kCountlyFBKeyShown, @"1",
-        kCountlyFBKeyWidgetID, self.ID];
+                             kCountlyQSKeySDKName, CountlyCommon.sharedInstance.SDKName,
```
Comment (arifBurakDemiray) [inline CountlyFeedbackWidget.m:143]: are those things related to formatter rigjt?

File: CountlyFeedbackWidget.m (hunk 2 above)
Comment (arifBurakDemiray) [reply CountlyFeedbackWidget.m:77]: if thid is related to formatter than no problemo

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
just couple of questions, ping me I will approve

**Review by arifBurakDemiray — APPROVED** [review]

File: CountlyFeedbackWidget.m (hunk 3 above)
Comment (ijunaid) [reply CountlyFeedbackWidget.m:143]: yes, most of the changes are related to auto formatting

**Comment by ArtursKadikis** [comment]
styling garbage seems to be added which should be cleaned up, @ijunaid

**Review by turtledreams — APPROVED** [review]

**Pull request merged**